Source/WebKit/UIProcess/gtk/AcceleratedBackingStore.h

```diff
@@ -54,7 +54,7 @@ class AcceleratedBackingStore {
 #if USE(GTK4)
     virtual void snapshot(GtkSnapshot*) = 0;
 #else
-    virtual bool paint(cairo_t*, const WebCore::IntRect&) = 0;
+    virtual bool paint(PlatformPaintContextPtr, const WebCore::IntRect&) = 0;
```

Comment:
I think you forgot to include `BackingStore.h` here, so the GTK builders are failing because this translation unit is not seeing the `PlatformPaintContextPtr` type definition. Maybe the easiest is to add something like the following above:

```cpp
#if USE(CAIRO)
typedef struct _cairo cairo_t;
using PlatformPaintContextPtr = cairo_t*;
#endif
```

I think this should be enough to fix the GTK build issue.

Reply:
Doh! I thought `AcceleratedBackingStore` descended from `BackingStore` which is not the case. Think I'm just going to roll back the changes in that class.